cmd/mobile-security-service/main.go

```diff
@@ -54,14 +58,30 @@ func initLogger(level, format string) {
 	}
 }
 
+// Make a connection to the PostgreSQL database
+func connectDatabase(c config.Config) *sql.DB {
+	dbConn, err := db.Connect(c.DB.ConnectionString, c.DB.MaxConnections)
+
+	if err != nil {
+		panic("failed to connect to SQL database: " + err.Error())
+	}
+
+	if err := db.Setup(dbConn); err != nil {
+		panic("failed to perform database setup: " + err.Error())
+	}
+
+	return dbConn
+}
+
 // Invoke handlers, services and repositories here
-func initHandlers(e *echo.Echo, c config.Config) {
+func initHandlers(e *echo.Echo, c config.Config, dbConn *sql.DB) {
```

Comment:
It shows still not make too much sense. Why a func with the name initHandler is using the database?

Reply:
+1 `initHandlers` badly named at the least now

Reply:
Agreed that the function name is bad but this would probably require change in a separate PR. I think it could all be refactored into a function called `setup` or something similar.

@camilamacedo86 the reason `initHandlers` is using the database is because the handlers require the database to be injected into them.